Formula/hidapi.rb

```diff
@@ -13,7 +13,8 @@ class Hidapi < Formula
     sha256 cellar: :any, mojave:        "e9c2bec30d5d1e9e0f9f91c43510071ba17234cd968b33f161c56cbee23a4d8d"
   end
 
-  depends_on "autoconf" => :build
+  # autoconf 2.70 fails with: configure.ac:16: error: AC_CONFIG_MACRO_DIR can only be used once
```

Comment:
This should be reported upstream (but we can merge before then)

Reply:
See https://github.com/libusb/hidapi/issues/264#issuecomment-830914402

Reply:
I adjusted the comment.